### DISCUSSION TITLE
(Feature Request) Ability to rename tabs

### DISCUSSION THREAD

**Discussion by baggachipz** [ghostty-org/ghostty#4118] [Feature Requests, Ideas] [answered]
I keep several tabs open at once. I love the Cmd+number shortcut to move between them. It would be much easier to see which tab does what if I could rename what it says in the tab area. Thanks for an amazing app!

**Comment by acsetter** [comment, marked as answer]
Hey @baggachipz , theres already a issue related to this feature request: #2509

**Reply by baggachipz** [reply]
I searched but didn't find that one. Thanks!
